code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -93,12 +93,26 @@
 	color = null
 	applies_material_colour = 0
 
-// Leaving this in for the sake of compilation.
 /obj/structure/bed/chair/comfy
+	name = "comfy chair"
 	desc = "It's a chair. It looks comfy."
 	icon_state = "comfychair"
 	base_icon = "comfychair"
 
+/obj/structure/bed/chair/comfy/Initialize()
+	. = ..()

```

Comment:
```suggestion
/obj/structure/bed/chair/comfy/update_icon()
	..()
```

Setting these in initialize only lasts until the next cut and rebuild which makes the north-facing state still bad when rotating.